Ported PersistentParrots and TNTDoNotUpdate

DeadlyMC opened the pull request:
Tweaked BlockTNT.class to add TNTDoNotUpdate tweak and EntityPlayer.class for PersistentParrots.

gnembon commented:
Looks good to me. Is it still WIP? If it is, no problem - take your time. Also will the docs be visible / reference-able via github/on some other venue, based on these files, or it has to be compiled manually by someone downloading the carpet?

lekro commented:
Still WIP, but if you would prefer to have this split over multiple PRs, it can be merged right now as well.

The docs can be accessed directly with the `.rst` files, which render just like markdown on github, but for full functionality it has to be compiled with sphinx. readthedocs.org offers this compiling and hosting with sphinx freely. But in order to do that, I think you will have to do the setup procedure (creating an account, connecting to github, etc.) on your end because you own the repo. You can just "sign in with GitHub" there and import the repository.

The output appears like [this](https://pixelart.readthedocs.io/en/latest/usage.html).

DeadlyMC committed:
PersistentParrots and TNTDoNotUpdate!

Tweaked BlockTNT.class to add TNTDoNotUpdate tweak and EntityPlayer.class for PersistentParrots.

gnembon commented:
Please fix merge conflicts. I assume you have access to your branch so it would be a bit easier for you.

DeadlyMC commented:
I prolly need to create a new PR... :P Cuz i had to delete the old fork due to branching issues again :(
I'm sorry for that...
